docs/constants/navigation.test.js

```diff
@@ -0,0 +1,46 @@
+const navigation = require('./navigation');
```

Comment:
@byCedric remove this before merging, it only validates if the replacement does _exactly_ what we need.